src/modules/temporary-file-check/file.service.get-file-size.test.ts

```diff
@@ -0,0 +1,50 @@
+import { FileGetPropertiesResponse } from '@azure/storage-file-share';
+import { BadRequestException } from '@nestjs/common';
+import { MAX_FILE_SIZE } from '@ukef/constants';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import { when } from 'jest-when';
+
+import { DtfsStorageFileService } from '../dtfs-storage/dtfs-storage-file.service';
```

Comment:
Can we use `@ukef/modules` import please?

Reply:
Changed